eng/Versions.props

```diff
@@ -18,14 +16,13 @@
     see issue https://github.com/dotnet/arcade/issues/1114
   -->
   <PropertyGroup>
-    <MicrosoftDotNetApiCompatVersion>1.0.0-beta.18522.6</MicrosoftDotNetApiCompatVersion>
-    <MicrosoftDotNetCodeAnalysisVersion>1.0.0-beta.18522.6</MicrosoftDotNetCodeAnalysisVersion>
+    <MicrosoftDotNetApiCompatVersion>1.0.0-beta.18525.8</MicrosoftDotNetApiCompatVersion>
+    <MicrosoftDotNetCodeAnalysisVersion>1.0.0-beta.18525.8</MicrosoftDotNetCodeAnalysisVersion>
     <MicrosoftDotNetGenApiVersion>1.0.0-beta.18522.6</MicrosoftDotNetGenApiVersion>
```

Comment:
@alexperovich @mmitche looks like the property name update are also case sensitive we should try and fix that.

Reply:
Yeah that's fixed in that tests-PR I have open.